stacks_queues/stack.py

```diff
@@ -12,27 +12,36 @@ def push(self, element):
         """ Adds an element to the top of the Stack.
             Returns None
         """
-        pass
+        self.store.add_last(element)
 
     def pop(self):
         """ Removes an element from the top
             Of the Stack
             Raises a StackEmptyException if
             The Stack is empty.
-            returns None
+            returns the removed element
         """
-        pass
+        if self.empty():
+            raise StackEmptyException()
+        
+        return self.store.remove_last()
+        
 
     def empty(self):
         """ Returns True if the Stack is empty
             And False otherwise
         """
-        pass
+        return self.store.empty()
 
     def __str__(self):
         """ Returns the Stack in String form like:
             [3, 4, 7]
             Starting with the top of the Stack and
             ending with the bottom of the Stack.
         """
```

Comment:
✨ This does work, but you could also take advantage of the `str` method in the `LinkedList` class